Documentation/project-docs/profiling-api-status.md

```diff
@@ -0,0 +1,41 @@
+APIs Internally Microsoft Tested:
```

Comment:
I think we should add some text at the top to provide context here. Perhaps something like this:

The notes below will help you determine what to profiling APIs are safe to use. The .Net Core project started with the codebase from the desktop CoreCLR/Silverlight so all the profiler APIs present there are also present in the code here. However that doesn't automatically imply that they are all working or being actively tested right now. Our goal is to eventually have everything tested and working across all the supported OSes. As we make progress we'll document it here. If you want to use APIs that we haven't tested yet you are welcome to do so, but you need to do your own testing to determine if it works. If you do test APIs we haven't gotten to yet, we hope you'll add a note below in the Community Tested API section so that everyone can benefit.